memote/support/helpers.py

```diff
@@ -316,9 +316,8 @@ def find_biomass_reaction(model):
 
     biomass_met = []
     for met in model.metabolites:
-        if met.id.lower().startswith('biomass') or met.name.lower().startswith(
-            'biomass'
-        ):
+        if met.id.lower().startswith('biomass') \
```

Comment:
We can make this condition slightly more generic if that's useful.
```python
        if 'biomass' in met.id.lower() or (met.name is not None and 'biomass' in met.name.lower()):
```

Reply:
Fair enough.